keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -471,21 +494,29 @@ type PEMTrustAnchorsProjection struct {
 
 #### Volume Content Generation and Refresh
 
-For each pemTrustAnchors projected volume source in the pod spec:
-
-1. If the named ClusterTrustBundle exists, kubelet will write
-   spec.pemTrustAnchors from the ClusterTrustBundle into the designated file.
-
-2. If the named ClusterTrustBundle does not exists, kubelet will fail the volume
-   mount process and emit an event describing the error.
-
-The volume will not be successfully mounted (and thus the pod will not start)
-until all certificate files have been written to the container filesystem.
-
-Once the pod has started, Kubelet will continue to update the contents of files
-as the associated ClusterTrustBundles change.  If the ClusterTrustBundle backing
-a file gets deleted, kubelet will keep the file at the last-available content,
-periodically emitting an event documenting the problem.
+For each ClusterTrustBundle projected volume source in the pod spec:
```

Comment:
The nested lists below are not rendered correctly in the markdown preview.

Reply:
Fixed, thanks.